### PR TITLE
Prepare staking and node-operation docs for live

### PR DESCRIPTION
**Pull request by 10thfloor**
## Description
PR ahead of releasing Staking & Delegation and Node Operation sections.
This is going to expose the Core Contracts docs as well.  

Also replaces "Devnet" with "Testnet"

### CODE REVIEW

File: docs/content/staking/technical-overview.mdx [hunk 1]
```diff
@@ -357,32 +357,32 @@ This script requires no arguments.
...
+You can use the **Get total FLOW staked by node type**([SC.07](../../protocol/core-contracts/flow-id-table-staking/#getting-staking-info)) script
 with the following arguments:
 
 | Argument   | Type     | Description |
 |------------|----------|-------------|
 | **nodeType** | `UInt8` | The type of node to search for. |
 
 The source code for the staking contract and more transactions 
-can be found in the [Flow Core Contracts GitHub Repository](https://github.com/onflow/flow-core-contracts/tree/josh/staking).
+can be found in the [Flow Core Contracts GitHub Repository](https://github.com/onflow/flow-core-contracts).
```
Comment (10thfloor) [inline docs/content/staking/technical-overview.mdx:388]: Is this correct?

Comment (joshuahannan) [reply docs/content/staking/technical-overview.mdx:388]: yes thank you for catching that!

File: docs/content/token/index.mdx [hunk 2]
```diff
@@ -150,3 +129,7 @@ Once the transaction is sealed, an event is emitted and you will be able to read
 
 The Flow SDKs also allow polling for events using the Flow Access API,
 [for example using the Flow Go SDK](https://github.com/onflow/flow-go-sdk#querying-events).
+
+## How to Build with FLOW
+
+Read more about how to start building on [Flow Testnet](), or [Flow Mainnet]().
```
Comment (psiemens) [inline docs/content/token/index.mdx:135]: Do these have links yet?

**Review by psiemens — COMMENTED** [review]
Very nice cleanup!

File: docs/content/token/index.mdx (hunk 2 above)
Comment (10thfloor) [reply docs/content/token/index.mdx:135]: They do now!

Comment (10thfloor) [inline docs/content/token/index.mdx:135]:
```suggestion
Read more about how to start building on [Flow Testnet]() (Coming soon.), or [Flow Mainnet]() (Coming soon).
```

File: docs/content/concepts/flow-concepts/accessing-testnet.mdx [hunk 3]
```diff
@@ -78,9 +78,9 @@ Some of these examples
 
 Transactions will now require [fees](https://www.notion.so/token/concepts/) in order to complete.
 
-## Important Devnet Smart Contract Addresses
+## Important Testnet Smart Contract Addresses
 
-In order to access some of the contracts deployed to Flow Devnet, you can import them by using the following import addresses:
+In order to access some of the contracts deployed to Flow Testnet, you can import them by using the following import addresses:
 
 ```cadence
```
Comment (joshuahannan) [inline docs/content/concepts/flow-concepts/accessing-testnet.mdx:85]: Could we link to the protocol pages here instead of duplicating the addresses?

**Review by joshuahannan — APPROVED** [review]
Looks great! Just had one comment

**Pull request merged**